### PR TITLE
RavenDB-10380 bulk insert not working on full .NET

### PR DESCRIPTION
**Pull request by talweiss1982**

### CODE REVIEW

File: src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs [hunk 1]
```diff
@@ -101,7 +102,11 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                 var message = new HttpRequestMessage
                 {
                     Method = HttpMethod.Post,
-                    Content = _stream
+                    Content = _stream,
+                    Headers =
+                    {
+                        TransferEncoding = { new TransferCodingHeaderValue("chunked") }
```
Comment (ppekrol) [inline src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs:108]: I think that there is 'TransferEncodingChunked' property in Headers available. Maybe we should use it instead?

Comment (ayende) [reply src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs:108]: We didn't see that in the intellisense, possible 1.3 vs 2.0 issue?

Comment (ppekrol) [reply src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs:108]: Or R# intellisense, everything compiles on my end:

```csharp
                var message = new HttpRequestMessage
                {
                    Method = HttpMethod.Post,
                    Content = _stream,
                    Headers =
                    {
                        TransferEncodingChunked = true
                    }
                };
```

Comment (ayende) [reply src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs:108]: I'll make the change

**Pull request merged**